Sources/Profile-View.php

```diff
@@ -600,8 +600,8 @@ function showAlerts($memID)
 	if (!empty($_REQUEST['alert']))
 	{
 		$alert_id = (int) $_REQUEST['alert'];
-
-		$alert = array_pop(($alerts = fetch_alerts($memID, $alert_id)));
+		$alerts = $alerts = fetch_alerts($memID, $alert_id);
```

Comment:
I think we only need one `$alerts = ` on this line. 🙂

Reply:
weird how I didn't notice that..